app/admin/global_country_platform/locations_members.rb

```diff
@@ -0,0 +1,87 @@
+ActiveAdmin.register_page 'Global Cw Platform Locations Members' do
```

Comment:
could the pages for locations and locations members be merged, or would that require merging the importers?

Reply:
I divided them as we have different importers for Locations and LocationMembers. So yes, it would require merging the importers